### ISSUE TITLE
Git Clone in terraform-tools-gitops stopped working

### ISSUE THREAD

**Issue by triceam** [cloud-native-toolkit/software-everywhere#493]
[Recent changes](https://github.com/cloud-native-toolkit/git-client/commits/main) in `gitu` looks like they broke the gitops repo module.  I just re-ran our automated tests, and all tests are failing:
https://github.com/cloud-native-toolkit/terraform-tools-gitops/pull/103

<img width="830" alt="image" src="https://user-images.githubusercontent.com/974294/189020551-844dd5b8-afd4-4943-88ad-8e5248587d1d.png">

Automation fails with:
```
╷
│ Error: local-exec provisioner error
│ 
│   with module.gitops.module.gitops-repo.null_resource.repo,
│   on .terraform/modules/gitops.gitops-repo/main.tf line 32, in resource "null_resource" "repo":
│   32:   provisioner "local-exec" {
│ 
│ Error running command
│ '.terraform/modules/gitops.gitops-repo/scripts/create-repo.sh 'github.com'
│ 'seansund' 'tools-gitopsa' 'false' 'T29a7mstHcYLUTp9' 'true'': exit status
│ 1. Output: Checking for existing repo: seansund/Test/tools-gitopsa
│ Host is github.com. Using Github api
│ Creating repo: seansund/Test/tools-gitopsa 
│ Host is github.com. Using Github api
│ Creating Github repo: seansund/tools-gitopsa
│   Repo created: https://github.com/seansund/tools-gitopsa.git
│ Initializing repo: https://github.com/seansund/tools-gitopsa.git
│ Initializing repo - https://github.com/seansund/tools-gitopsa.git
│ Cloning Github repo: tools-gitopsa
│ Error cloning repo
│ 
╵
Error: Process completed with exit code 1.
```

**Comment by triceam** [comment]
I rolled back the `gitu` version in the latest `cli-tools-core` image until this is resolved:
https://github.com/cloud-native-toolkit/image-cli-tools-core/pull/29

This will propagate to all cli-tools image variants.

**Comment by triceam** [comment]
Any user that is using the `terraform` image, or or using manual download will still be impacted.

**Comment by seansund** [comment]
Fixed with gitu v1.14.4
